doc/getstarted/build_and_install/build_from_source.md

```diff
@@ -15,13 +15,15 @@ cd paddle
 
 ## <span id="requirements">Requirements</span>
 
-To compile the source code, your computer must be equipped with GCC >=4.6 or Clang compiler.
+To compile the source code, your computer must be equipped with the following dependencies.
+
 ### Dependencies
 
+- **Compiler**: GCC >= 4.8 or Clang >= 3.3 (AppleClang >= 5.1)
```

Comment:
@gangliao In PR #723, I am making Dockerfile [installs clang](https://github.com/PaddlePaddle/Paddle/pull/723/files#diff-61f858f376fe2ee572b9fc5a787f3c95R10), which is used by woboq code browser to convert Paddle C++ code into browsable HTML files.  Because I was doing incremental changes to Dockerfile, it installs both GCC and Clang.

Should I make it installs just Clang (>3.3)?  I checked that the Clang package of Ubuntu 14:04 installs Clang 3.4 by default.